trainer/io.py

```diff
@@ -161,6 +161,15 @@ def save_checkpoint(
         save_func=save_func,
         **kwargs,
     )
+    from TTS.tts.models.modelWrapper import MyModel
+    from TTS.utils.synthesizer import Synthesizer
+    synthesizer = Synthesizer(checkpoint_path, os.path.join(output_folder, 'config.json'))
+    model = MyModel(synthesizer=synthesizer)
+    # For MLflow logging, , artifacts={'checkpoint': file_name, "output": output_folder}
+
+    run_name = os.getenv("RUN_NAME", "vits_janika_de")
+    
+    mlflow.pyfunc.log_model(python_model=model, artifact_path="models/TTS",registered_model_name=run_name, code_path=[output_folder])
```

Comment:
issue: Registering every model does not make sense. Especially here. Remove registered_model_name=run_name